specification/attributes/discount_handling.md

```diff
@@ -0,0 +1,44 @@
+# Discount Handling
+
+A discount is a type of pricing model in which cloud providers offer their customers a reduced price that is applied to the usage of the cloud provider’s services. There are many types of discounts that a cloud provider can support, including but not limited to commercially negotiated discounts, commitment-based discounts that offer lower prices when you agree to a certain amount of their services or spend, and bundled discounts where you receive free or discounted usage of one product or service based on the usage of another product or service. Discount handling is commonly used in scenarios for verifying discounts were applied and calculating cost savings.
```

Comment:
Should we avoid "pricing model" given there's another column or did you want to reference that intentionally? Also generalizing to support discounts for purchases and suggesting a few ways to shorten this a bit...
```suggestion
A discount is a pricing construct where providers offer a reduced price for services. Providers may have many types of discounts, including but not limited to commercially negotiated discounts, commitment-based discounts when you agree to a certain amount of usage or spend, and bundled discounts where you receive free or discounted usage of one product or service based on the usage of another. Discount Handling is commonly used in scenarios like verifying discounts were applied and calculating cost savings.
```